lib/build-all-ng.sh

```diff
@@ -72,7 +73,7 @@ pack_upload ()
 	# pack and upload to server or just pack
 
 	display_alert "Signing" "Please wait!" "info"
-	local version="Armbian_${REVISION}_${BOARD^}_${RELEASE}_${BRANCH}_${VER/-$LINUXFAMILY/}${DESKTOP_ENVIRONMENT:+_$DESKTOP_ENVIRONMENT}"
+	local version="${VENDOR}_${REVISION}_${BOARD^}_${RELEASE}_${BRANCH}_${VER/-$LINUXFAMILY/}${DESKTOP_ENVIRONMENT:+_$DESKTOP_ENVIRONMENT}"
```

Comment:
If VENDOR is not defined it should be "Armbian" ?

Reply:
Sorry... 
I put initialization and default value into configuration.sh, but it is too late for build-all-ng.sh.
I removed VENDOR from unset_all () function, because it may be problem for build-all-ng.sh too.

I have not use build-all-ng.sh, and I can't test it properly.